src/utils/address_test.cljs

```diff
@@ -0,0 +1,42 @@
+(ns utils.address-test
+  (:require [cljs.test :refer [deftest is testing]]
+            [utils.address]))
+
+(deftest get-shortened-compressed-key
+  (testing "Check if the function returns the correct shortened key"
+    (is (= "zQ3...1sgt5N"
+           (utils.address/get-shortened-compressed-key
+            "zQ3ssgRy5TtB47MMiMKMKaGyaawkCgMqqbrnAUYrZJ1sgt5N"))))
+
+  (testing "Check if the function handles empty string"
+    (is (nil? (utils.address/get-shortened-compressed-key ""))))
+
+  (testing "Check if the function handles nil"
+    (is (nil? (utils.address/get-shortened-compressed-key nil))))
+
+  (testing "Check if the function handles keys shorter than 9 characters"
+    (is (nil? (utils.address/get-shortened-compressed-key "abc")))
+    (is (nil? (utils.address/get-shortened-compressed-key "1234")))))
+
+
+(deftest test-get-abbreviated-profile-url
+  (testing "Check if the function returns the correct abbreviated profile URL"
+    (is (= "join.status.im/u/zQ3sh...aimrdYpzeFUa"
+           (utils.address/get-abbreviated-profile-url
+            "join.status.im/u/"
+            "zQ3shPrnUhhR42JJn3QdhodGest8w8MjiH8hPaimrdYpzeFUa")))
+
+    (testing "Check if the function handles empty public key"
```

Comment:
I think these test descriptions could be clearer.
e.g
"Check value is nil if the function input is empty public key"